src/accessibility/outputs.js

```diff
@@ -5,18 +5,20 @@
  * @requires core
  */
 
-// Errors when running example:
-// - point
-// - line
-// - appending does not work
-// Call update only at the end of draw?
+// To-do:
+// Improve updating to include only shapes that have changed? [leave as an issue]
```

Comment:
thinking about this more since we last discussed, this requires:
- updating items that are a match
- removing items that are no longer present
- adding new items